administrator-documentation/moderne-platform/how-to-guides/dev-center.md

```diff
@@ -0,0 +1,329 @@
+
+{% hint style="success" %}
+When creating a DevCenter for the first time, we **strongly recommend** that you only create a DevCenter for a few key organizations. This will allow you to get data into the platform faster and ensure that you've configured everything correctly. Once everything is working as expected, you can then add more DevCenters as desired.
+{% endhint %}
+
+### Upgrade and migration cards
+
+Upgrade and migration cards allow you to see things like what Java version your repositories use, what version of Spring Boot they're on, or what JUnit version they use:
+
+![](/.gitbook/assets/migration-card.png)
+
+To create these cards, you will need three things:
+
+TODO: ADD LINKS TO THE BELOW BULLETS
+
+1. A relevant title for the card (e.g., `Spring Boot 3.2`)
+2. A recipe that can be run to fix the core issue the card is highlighting (e.g., `org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_2`)
+3. Some measures that break up the card into key categories and help your users determine a level of urgency. 
+
+#### Measures
+
+Each measure consists of a name and a recipe that can be used to determine whether or not a repository falls into said measure. This recipe _should not_ be the same recipe as the one used to "fix" the card. For instance, in a `Spring Boot 3.2` card, one measure might have a name of `Major` – which represents all repositories that are one or more major versions behind. The corresponding recipe would be `org.openrewrite.java.dependencies.DependencyInsight` and it would include the options of `1-2`. This would mark all repositories that use Spring Boot version `1.x` or `2.x` as being a major version or more behind.
+
+Each card can have up to **three measures**. These measures should be returned in a specific order; with the most urgent being returned first and the least urgent being returned last. In the `Spring Boot 3.2` example, you might have: `Major`, `Minor`, and `Patch` returned in that specific order.
+
+When determining what repositories fall into what measure, Moderne will first check all repositories against the most urgent measure. If any of them are marked by that first recipe, they will be added to that measure and not checked again. The second most urgent measure will then be checked, and the same thing will happen. Lastly the third most urgent measure will be checked. If any repositories have no results for _all_ of the measures, they are considered to be "Completed" and will be marked as such on the card.
+
+{% hint style="warning" %}
+If you change any part of any measure on a card (such as the name of the measure or what recipe it should run), you will lose all results for the card until the next rebuild of the DevCenter.
+{% endhint %}
+
+#### Example
+
+<details>
+
+<summary>
+Below is an example of an upgradesAndMigrations section for a Spring Boot 3.2 card:
+</summary>
```

Comment:
For later: we're going to want a document with a catalog of recommended measures for various upgrades and migrations.